packages/site/src/routes/[dictionaryId]/entry/EntryDialect.svelte

```diff
@@ -0,0 +1,34 @@
+<script lang="ts">
+  import { _ } from 'svelte-i18n';
+  import ShowHide from 'svelte-pieces/functions/ShowHide.svelte';
+  import type { IEntry } from '@living-dictionaries/types';
+  export let entry: IEntry,
+    canEdit = false;
+  let value: string;
+  $: value = entry.di;
+</script>
+
+{#if value || canEdit}
+  <ShowHide let:show let:set let:toggle>
+    <div
+      class="md:px-2 rounded"
+      on:click={() => set(canEdit)}
+      class:hover:bg-gray-100={canEdit}
+      class:cursor-pointer={canEdit}
+      class:order-2={!value}>
+      <div class="text-xs text-gray-500 mt-1">{$_('entry.di', { default: 'Dialect' })}</div>
+      <div class="border-b-2 border-dashed pb-1 mb-2">
+        {#if value}
+          <div dir="ltr">
+            {value}
+          </div>
+        {:else}<i class="far fa-pencil text-gray-500 text-sm" />{/if}
+      </div>
+    </div>
+    {#if show}
+      {#await import('$lib/components/modals/DialectModal.svelte') then { default: DialectModal }}
```

Comment:
@Danble in the same line where you lazy load components please destructure the default export to the component name so you don't have to write <DialectModal.default> - this makes the code easier to copy-paste into places where components aren't lazy-loaded.